ui/apps/platform/src/services/GroupsService.js

```diff
@@ -23,6 +23,19 @@ export function updateOrAddGroup({ oldGroups, newGroups }) {
     return axios.post(updateUrl, { previous_groups: oldGroups, required_groups: newGroups });
 }
 
+/**
+ * Fetches the default rule.
+ *
+ * @returns {Promise<Object, Error>} fulfilled with array of groups
+ */
+export function getDefaultGroup({ authProviderId, roleName }) {
+    return axios
+        .get(`${url}?authProviderId=${authProviderId}&key=&value=&roleName=${roleName}`)
```

Comment:
Does the API require us to spend a `key` and `value` with blank values specifically?
Seems unnecessary--is it a limitation of the proto system?

Reply:
I thought that 'empty' and 'absence' will map to the same logical situation: don't filter by this field.

Reply:
No, surprisingly not.

Say you have two groups:
```json
{"props":{"authProviderId":"0ceed22f-026f-41a2-9d91-96410e4c4846","key":"","value":""},"roleName":"Scope Manager"}
{"props":{"authProviderId":"0ceed22f-026f-41a2-9d91-96410e4c4846","key":"email","value":"something"},"roleName":"Scope Manager"}
```

both groups are mapping to the role name `Scope Manager`.

Executing a request, i.e. via `curl`, you will retrieve the following **without explicit empty fields**:
```
curl <endpoint>/v1/groups?authProviderId=0ceed22f-026f-41a2-9d91-96410e4c4846&roleName="Scope Manager"

response:
{"groups":[{"props":{"authProviderId":"0ceed22f-026f-41a2-9d91-96410e4c4846","key":"","value":""},"roleName":"Scope Manager"},{"props":{"authProviderId":"0ceed22f-026f-41a2-9d91-96410e4c4846","key":"email","value":"something"},"roleName":"Scope Manager"}]}
```

Instead, if you execute the request with **explicitly setting key and value to empty strings**, you will receive the following:
```
curl <endpoint>/v1/groups?authProviderId=0ceed22f-026f-41a2-9d91-96410e4c4846&roleName="Scope Manager"&key=&value=

response:
{"groups":[{"props":{"authProviderId":"0ceed22f-026f-41a2-9d91-96410e4c4846","key":"","value":""},"roleName":"Scope Manager"}]}
```

TL;DR: we do need to explicitly set those, otherwise we return other groups that are not the default created one.